packages/jss/tests/integration/rules.js

```diff
@@ -388,6 +388,11 @@ describe('Integration: rules', () => {
         `)
       })
     })
+
+    it('should uglify CSS', () => {
+      const rule = jss.createRule('a', {float: 'left', width: '1px'})
+      expect(rule.toString({uglify: true})).to.be('.a-id {float: left;width: 1px;}')
```

Comment:
it could be even float:left;width:1px;

Reply:
and the space between selector and curly brace is not needed either

Reply:
done in 2023d19